devel/gaa/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                github 1.0
+
+github.setup             cooljeanius gaa 1.6.6_1 v
+revision                 3
+platforms                darwin
+categories               devel
+maintainers              gwmail.gwu.edu:egall openmaintainer
```

Comment:
There should be a Github handle here as well so that the maintainer can get a notification on PRs against this port.

Reply:
Added in [3e0470f](https://github.com/macports/macports-ports/pull/20972/commits/3e0470fc5ad30184fec83c23722be7b056bbd219).